packages/main/src/themes/Label.css

```diff
@@ -20,6 +20,11 @@
 	overflow: hidden;
 }
 
+bdi{
+    content: "";
+	padding-right: 2.5px;
```

Comment:
Should the bdi also gets its RTL mirrored padding?
And in general all sizes is good to be in rems

Reply:
No it is not needed.
Size is in rems now